include/Layer/Core/VBound.hpp

```diff
@@ -26,30 +26,30 @@
 
 namespace VGG
 {
-class Bound
+class Bounds
```

Comment:
Why change `Bound` to `Bounds`? If it is necessary, why not change functions namely accordingly, for example, `PaintNode::setFrameBound`. There are plenty of name mismatches.

Reply:
As well as member data, like `VNode::m_bound`.